its/src/test/java/com/sonar/maven/it/suite/JavaTest.java

```diff
@@ -26,7 +26,9 @@
 import java.io.FileInputStream;
 import java.io.IOException;
 import java.nio.file.Path;
+import java.util.Arrays;
 import java.util.Properties;
+import org.junit.jupiter.api.Assumptions;
```

Comment:
We can remove this lefover imports from previous iterations
```suggestion
import java.util.Properties;
```

Reply:
:+1: